resources/quality/katihal/alya3dp_normal_flex.inst.cfg

```diff
@@ -0,0 +1,34 @@
+[general]
+version = 4
+definition = alya3dp
+name = Flex Normal
+
+[metadata]
+setting_version = 5
+type = quality
+quality_type = alya_normal
+weight = 2
+material = alya_flex
+
+[values]
+material_diameter = 1.75
```

Comment:
This seems like a typical setting to put in the material profile...

Reply:
I thought it might be a good idea to add this just to be sure, as material settings are overriden by the print profile. Leaving this on would make material changes etc safer, in my humble opinion. Our extruder wouldn't support 3mm anyways. Does it have any other negative consequences levaing this line here?

Reply:
There was a profile for flex materials, which i accidently left there, but which has no corresponding material definition in th ematerial database yet.

Reply:
I deleted alya_flex printing profiles